client/rest-high-level/src/test/java/org/elasticsearch/client/RequestConvertersTests.java

```diff
@@ -1225,10 +1225,12 @@ public void testMultiSearchTemplate() throws Exception {
 
     public void testExplain() throws IOException {
         String index = randomAlphaOfLengthBetween(3, 10);
-        String type = randomAlphaOfLengthBetween(3, 10);
+        String type = randomBoolean() ? null : randomAlphaOfLengthBetween(3, 10);
```

Comment:
let's test both cases all the time?

Reply:
Will do, that would've helped catch the issue above.